features/CollectionController.feature

```diff
@@ -38,6 +38,29 @@ Feature: Collection Controller
     Then I should receive a "hits" array of objects matching:
       | _id          | _source      |
       | "document-1" | { "age": 2 } |
+  
+  Scenario: Update a collection and search document (dyanmic false)
+    Given an index "nyc-open-data"
+    And I "create" the collection "nyc-open-data":"green-taxi" with:
+      | mappings | { "dynamic": "false", "properties": { "name": { "type": "keyword" }, "metadata": {"properties": {}, "dynamic": "false"} } } |
+    And I "create" the following documents:
+      | _id          | body        |
+      | "document-1" | {"age": 2} |
+    And I refresh the collection
+    When I "update" the collection "nyc-open-data":"green-taxi" with:
+      | mappings | { "properties": { "age": { "type": "long" } } } |
+    When I search documents with the following query:
+      """
+      {
+        "match": {
+          "age": 2
+        }
+      }
+      """
```

Comment:
It's good but you also need to verify that the search does not match the field `age` before updating the mapping and triggering the reindex